sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -554,10 +554,10 @@ namespace Azure { namespace Core { namespace Http { namespace _detail {
    * multiple times based on the state of the TLS connection.
    *
    * Special consideration for the WINHTTP_CALLBACK_STATUS_SENDING_REQUEST - this callback is
-   * called during the TLS connection - if a TLS root certificate is configured, we verify that the
-   * certificate chain sent from the server contains the certificate the HTTP client was configured
-   * with. If it is, we accept the connection, if it is not, we abort the connection, closing the
-   * incoming request handle.
+   * called during the TLS connection - if a TLS root certificate is configured, we verify that
+   * the certificate chain sent from the server contains the certificate the HTTP client was
+   * configured with. If it is, we accept the connection, if it is not, we abort the connection,
+   * closing the incoming request handle.
    */
```

Comment:
What's causing these comment/line break changes? They are unrelated, correct?

Reply:
I had to run clang-format on the file since I added a comment.  Clang-format did them.  

The change is still more than 1 line b/c I had to add comments.  I'd rather leave these comment fixes as they are correct and are only comments.